### PR TITLE
Review B3 - animated silverware swarm, furnace, trebuchet, colossus

### PR DESCRIPTION
**Pull request by MSAbaddon**

### CODE REVIEW

File: packs/data/pathfinder-bestiary-3.db/animated-colossus.json [hunk 1]
```diff
@@ -29,8 +29,11 @@
             "allSaves": {
                 "value": ""
             },
+            "hardness": {
+                "value": 15
+            },
             "hp": {
-                "details": "Hardness 15",
+                "details": "",
```
Comment (TikaelSol) [inline packs/data/pathfinder-bestiary-3.db/animated-colossus.json:36]: ?

File: packs/data/pathfinder-bestiary-3.db/animated-trebuchet.json [hunk 2]
```diff
@@ -29,8 +29,11 @@
             "allSaves": {
                 "value": ""
             },
+            "hardness": {
+                "value": 14
+            },
             "hp": {
-                "details": "Hardness 14",
+                "details": "",
```
Comment (TikaelSol) [inline packs/data/pathfinder-bestiary-3.db/animated-trebuchet.json:36]: Same thing, did we decide to not add hardness in here?

File: packs/data/pathfinder-bestiary-3.db/animated-colossus.json (hunk 1 above)
Comment (MSAbaddon) [reply packs/data/pathfinder-bestiary-3.db/animated-colossus.json:36]: Well, NPCs do have their own hardness field now, so I entered the value there:
image

Comment (TikaelSol) [reply packs/data/pathfinder-bestiary-3.db/animated-colossus.json:36]: Ah, yes.

**Pull request merged**